src/include/codegen/bloom_filter.h

```diff
@@ -0,0 +1,75 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// bloom_filter.h
+//
+// Identification: include/codegen/bloom_filter.h
+//
+// Copyright (c) 2015-2017, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include <vector>
+
+#include "codegen/codegen.h"
+#include "codegen/hash.h"
+
+namespace peloton {
+namespace codegen {
+
+class BloomFilter {
```

Comment:
In general all the C++ classes that manage memory go in `codegen/util` and the "accessor" classes go in `codegen`. Can you extract these classes into two?